.github/workflows/ci.yml

```diff
@@ -38,6 +38,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       CARGO_BUILD_TARGET: ${{ matrix.target }}
+      CARGO_BINSTALL_LOG_LEVEL: debug
```

Comment:
could we set this to debug only if the action is in debug mode, so the logs are brief in the happy case?

Reply:
indeed the justfile already does similarly so

Reply:
Yeah because I forgot `BINSTALL_LOG_LEVEL` exists.
I clearly saw this when reviewing your PR but I guess we should document these behavior.